client/src/components/Feed/Post.js

```diff
@@ -842,18 +843,26 @@ const Post = ({
     </>
   );
 };
-const renderContent = (title, content, highlightWords, showComplete) => {
+const renderContent = (title, content, highlightWords, showComplete, linkifyText) => {
   let finalContent = content;
   if (finalContent.length > CONTENT_LENGTH && !showComplete) {
     finalContent = `${finalContent.substring(0, CONTENT_LENGTH)} . . .`;
   }
   return (
     <Card.Body className="content-wrapper">
       <Heading level={4} className="h4">
-        <Highlight textObj={linkify(title)} highlight={highlightWords} />
+      {
+        linkifyText ? <Highlight textObj={linkify(title)} highlight={highlightWords} /> :
```

Comment:
Hi @aditilamkhede , could this be simplified as `<Highlight textObj={linkifyText ? linkify(title) : title} highlight={highlightWords} />` ?